tests/integration/011_policy/main.tf

```diff
@@ -18,17 +18,19 @@ resource "env0_project_policy" "test_policy" {
   disable_destroy_environments  = false
   skip_redundant_deployments    = false
   drift_detection_cron          = var.second_run ? "0 4 * * *" : "0 3 * * *"
+
 }
 
 resource "env0_project_policy" "test_policy_2" {
-  project_id                    = env0_project.test_project.id
-  number_of_environments        = 1
-  number_of_environments_total  = 1
-  requires_approval_default     = true
-  include_cost_estimation       = true
-  skip_apply_when_plan_is_empty = true
-  disable_destroy_environments  = true
-  skip_redundant_deployments    = true
+  project_id                      = env0_project.test_project.id
+  number_of_environments          = 1
+  number_of_environments_total    = 1
+  requires_approval_default       = true
+  include_cost_estimation         = true
+  skip_apply_when_plan_is_empty   = true
+  disable_destroy_environments    = true
+  skip_redundant_deployments      = true
+  vcs_pr_comments_enabled_default = true
```

Comment:
This tests the project policy configuration?

Reply:
just sanity really.
The better test is the acceptance testing.